azure-pipelines.yml

```diff
@@ -98,41 +120,11 @@ stages:
         MAX_WORKER: $(T0_2VLANS_INSTANCE_NUM)
         DEPLOY_MG_EXTRA_PARAMS: "-e vlan_config=two_vlan_a"
 
-  - job:
-    pool: ubuntu-20.04
-    displayName: "kvmtest-t0"
```

Comment:
Why remove the t0 and t1-lag check?

Reply:
t0 and t1-lag check contain classic and TBv2 tests, they were used to help pass PR test when TBv2 checks were not stable enough. But now, TBv2 checks are stable and classic tests are skipped, so we no longer need these 2 checks.

Reply:
Then classic tests should be removed along with these parent checks depends on both classic and testbedv2 checks, right?

Reply:
We can keep the classical test jobs, they're independent of testbedV2 test jobs, and any one of them passes can be deemed as a pass.
We had the parent t0 and t1-lag because, in the past, the pr only checked the results of t0 and t1-lag, but now the pr checks the whole job(Azure.sonic-mgmt), so we don't need to keep them anymore.